test/tests/api/v1_1/os_install_tests.py

```diff
@@ -1,27 +1,162 @@
 from config.api1_1_config import *
-from obm_settings import obmSettings
+from config.settings import *
+from modules.logger import Log
 from on_http_api1_1 import NodesApi as Nodes
 from on_http_api1_1 import rest
-from modules.logger import Log
+from workflows_tests import WorkflowsTests as workflows
 from datetime import datetime
 from proboscis.asserts import assert_equal
 from proboscis.asserts import assert_false
 from proboscis.asserts import assert_raises
 from proboscis.asserts import assert_true
 from proboscis.asserts import assert_not_equal
+from proboscis.asserts import assert_is_not_none
 from proboscis import SkipTest
 from proboscis import test
+from proboscis import after_class
+from proboscis import before_class
 from json import dumps, loads
+import os
 
 LOG = Log(__name__)
 
-@test(groups=['os-install.v1.1.tests'])
+@test(groups=['os-install.v1.1.tests'], \
+    depends_on_groups=['amqp.tests'])
 class OSInstallTests(object):
 
     def __init__(self):
-        self.__client = config.api_client 
+        self.__client = config.api_client
+        self.__base = os.getenv('RACKHD_BASE_REPO_URL', \
+            'http://{0}:{1}'.format(HOST_IP, HOST_PORT))
+            
+    @before_class()
+    def setup(self):
+        pass
+        
+    @after_class(always_run=True)
+    def teardown(self):
+        self.__format_drives()  
+    
+    def __get_data(self):
+        return loads(self.__client.last_response.data)
+    
+    def __post_workflow(self, graph_name, nodes, body):
+        workflows().post_workflows(graph_name, timeout_sec=3600, nodes=nodes, data=body)         
+
+    def __format_drives(self):
```

Comment:
@jlongever , but the OS-bootstrap will always depend on there is ipmi-obm-service, otherwise it will fail. If ipmi-obm-service is existing, the OS-boostrap will  temporary set set boot-order to pxe regardless the BIOS persistent boot order setting.